lib/datasource/remote/model/firebase_models/region_location_model.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+class RegionLocation {
+  final String regionAccount;
```

Comment:
this is the region ID right?

Reply:
yes, but chain calls it regionAccount, I wanted to keep consistent